Controller/UploadController.php

```diff
@@ -33,10 +35,15 @@ public function uploadImageAction(Request $request
         
         $webDir = $config['uploadConfig']['webDir'];
         $webDir = substr($webDir, -strlen('/')) === '/' ? $webDir : $webDir . '/';
-        $filename = sha1(uniqid(mt_rand(), true));
-        
-        $thumbsDir = $this->container->getParameter('comur_image.thumbs_dir');
-        $thumbSize = $this->container->getParameter('comur_image.media_lib_thumb_size');
+        if($config['uploadConfig']['generateFilename']) $filename = sha1(uniqid(mt_rand(), true));
```

Comment:
Hi @epidoux 

There is just this one left, thanks! And thanks for your help anyway !